clang/tools/dpct/api_migration_status_arrays.py

```diff
@@ -0,0 +1,23 @@
+"""
+module: arraies of strings of unsupports CUDA APIs converted from the database of SYCLomatic project
+
+The database of CUDA migration APIs status is stored under https://github.com/oneapi-src/SYCLomatic/tree/SYCLomatic/docs/dev_guide/api-mapping-status
+
+"""
+__updatetodate__ = "20230926"
+
+ASM_API_migration_status = ['activemask', 'addc', 'alloca', 'applypriority', 'atom', 'bar', 'barrier', 'bfe', 'bfi', 'bfind', 'bmsk', 'bra', 'brev', 'brkpt', 'brx', 'call', 'copysign', 'cos', 'cp', 'createpolicy', 'cvt', 'cvta', 'discard', 'dp2a', 'dp4a', 'elect', 'ex2', 'exit', 'fence', 'fma', 'fns', 'getctarank', 'griddepcontrol', 'isspacep', 'istypep', 'ld', 'ldmatrix', 'ldu', 'lg2', 'madc', 'mapa', 'match', 'mbarrier', 'membar', 'mma', 'movmatrix', 'multimem', 'nanosleep', 'pmevent', 'prefetch', 'prefetchu', 'prmt', 'rcp', 'red', 'redux', 'ret', 'rsqrt', 'sad', 'selp', 'set', 'setmaxnreg', 'shf', 'shfl', 'sin', 'slct', 'sqrt', 'st', 'stackrestore', 'stacksave', 'stmatrix', 'subc', 'suld', 'suq', 'sured', 'sust', 'szext', 'tanh', 'testp', 'tex', 'tld4', 'trap', 'txq', 'vabsdiff2', 'vabsdiff4', 'vadd2', 'vadd4', 'vavrg2', 'vavrg4', 'vmad', 'vmax2', 'vmax4', 'vmin2', 'vmin4', 'vote', 'vset', 'vset2', 'vset4', 'vshl', 'vshr', 'vsub2', 'vsub4', 'wgmma', 'wmma']
```

Comment:
Suggest using the python script to scan the APINames_*.inc files in https://github.com/oneapi-src/SYCLomatic/tree/SYCLomatic/clang/lib/DPCT folder to get the latest the migration status of api and asm intrinsic, so that we can avoid maintaining the copies of the migration status of api and asm intrinsic here.